init.lua

```diff
@@ -188,8 +196,13 @@ local function delayed_armor_check(pname)
 		-- this can fail inside register_on_joinplayer.
 		local elms=armor:get_weared_armor_elements(player)
 		if elms == nil then
-			-- If it does fail, wait another second (it's possible to get into an infinite loop here...)
-			minetest.log("info", "ediblestuff: armor check was delayed...")
+			-- Prevent log flooding
```

Comment:
Out of curiosity, was this actually logging at all for you?

Reply:
Only once:
2022-08-19 00:39:31: INFO[Server]: ediblestuff: armor check was delayed...

It's mostly before of the "infinite loop" comment. I've seen this often being an issue in my line of work.